src/buffer/out/textBuffer.hpp

```diff
@@ -131,7 +131,7 @@ class TextBuffer final
 
     const COORD GetWordStart(const COORD target, const std::wstring_view wordDelimiters, bool accessibilityMode = false) const;
     const COORD GetWordEnd(const COORD target, const std::wstring_view wordDelimiters, bool accessibilityMode = false) const;
-    bool MoveToNextWord(COORD& pos, const std::wstring_view wordDelimiters, COORD lastCharPos) const;
+    bool MoveToNextWord(COORD& pos, const std::wstring_view wordDelimiters, const COORD lastCharPos) const;

```

Comment:
surprised this compiles

Reply:
derp. Just added it to the cpp too.

Reply:
The correct way to fix this would be to back out _all changes to `MoveToNextWord`_